examples/seismic/viscoacoustic/viscoacoustic_example.py

```diff
@@ -38,11 +38,11 @@ def run(shape=(50, 50), spacing=(20.0, 20.0), tn=1000.0,
 
 @pytest.mark.parametrize('kernel, time_order, normrec, atol', [
     ('sls', 2, 684.385, 1e-2),
-    ('sls', 1, 18.774, 1e-2),
```

Comment:
what is the main reason of this increase in norms?

Reply:
Just because I reorganized the equations and the source scale changed to` scale = dt / m`